cloudflare/resource_cloudflare_ruleset_test.go

```diff
@@ -1260,6 +1260,47 @@ func TestAccCloudflareRuleset_ExposedCredentialCheck(t *testing.T) {
 	})
 }
 
+func TestAccCloudflareRuleset_Logging(t *testing.T) {
```

Comment:
this fails locally due to a bad `kind`

```
TF_ACC=1 go test $(go list ./...) -v -run "^TestAccCloudflareRuleset_Logging" -count 1 -parallel 1 -timeout 120m -parallel 1
?   	github.com/cloudflare/terraform-provider-cloudflare	[no test files]
=== RUN   TestAccCloudflareRuleset_Logging
=== PAUSE TestAccCloudflareRuleset_Logging
=== CONT  TestAccCloudflareRuleset_Logging
    resource_cloudflare_ruleset_test.go:1279: Step 1/1 error: Error running apply: exit status 1

        Error: error creating ruleset example disable logging for skip rule: 'zone' is not a valid value for Kind because kind "zone" can only be used for zone level rulesets.

          with cloudflare_ruleset.gyuiknnwgs,
          on terraform_plugin_test.tf line 2, in resource "cloudflare_ruleset" "gyuiknnwgs":
           2:   resource "cloudflare_ruleset" "gyuiknnwgs" {

--- FAIL: TestAccCloudflareRuleset_Logging (11.04s)
FAIL
FAIL	github.com/cloudflare/terraform-provider-cloudflare/cloudflare	11.549s
?   	github.com/cloudflare/terraform-provider-cloudflare/tools/cmd/changelog-check	[no test files]
?   	github.com/cloudflare/terraform-provider-cloudflare/tools/cmd/maintainer-only-file-check	[no test files]
?   	github.com/cloudflare/terraform-provider-cloudflare/tools/cmd/tf-log-check	[no test files]
?   	github.com/cloudflare/terraform-provider-cloudflare/version	[no test files]
FAIL
make: *** [testacc] Error 1
```

Reply:
Oups :sweat_smile: Just pushed a fix!